### ISSUE TITLE
Training is ineffective

### ISSUE THREAD

**Issue by imfaimt** [swimmiing/ACL-SSL#7]
Hello, author! I'm reproducing your original code. I installed the environment according to your documentation. I trained with batch size 20 and a learning rate of 0.0004. The audio was 16 kHz and the frame rate was 120. However, the results are inconsistent.

I'd like to ask where I went wrong? The article says the Adam learning rate is 0.001, but the code says it's 0.0001. Which one is correct?

<img width="682" height="1240" alt="Image" src="https://github.com/user-attachments/assets/051c8cb8-6af9-43ea-9e9b-16963cc29b45" />

<img width="1306" height="1004" alt="Image" src="https://github.com/user-attachments/assets/67b0d663-7871-4131-a9f9-ab7f014da534" />

**Comment by swimmiing** [comment]
Hi, thanks for your interest in reproducing our work!

As mentioned in https://github.com/swimmiing/ACL-SSL/issues/2, there was a typo in the paper regarding the learning rate, please use the value from the released code instead. It has been corrected in our IJCV journal paper https://arxiv.org/abs/2505.05343

Also, we recommend training with 2 GPUs and a batch size of 16. Depending on the combination of batch size and learning rate, training can sometimes be unstable, but with this configuration, it should work properly.

Hope this helps!
